looks like some test s failing

/../../../usr/share/miniconda/lib/python3.11/site-packages/pkg_resources/__init__.py:2350
  /usr/share/miniconda/lib/python3.11/site-packages/pkg_resources/__init__.py:2350: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('cloudmesh')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(parent)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_base.py::Test_Base::test_environment_variable_path - AssertionError: assert '/home/runner/.cloudmesh' == './tmp/.cloudmesh'
  - ./tmp/.cloudmesh
  + /home/runner/.cloudmesh
FAILED tests/test_ping.py::Test_ping::test_internal_ping - assert False
FAILED tests/test_ping.py::Test_ping::test_ping_processor - assert False
FAILED tests/test_shell.py::Test_shell::test_pwd - AssertionError: assert ('cm/cloudmesh-common' in '/home/runner/work/cloudmesh-common/cloudmesh-common' or 'cm\\cloudmesh-common' in '/home/runner/work/cloudmesh-common/cloudmesh-common')
FAILED tests/test_shell_tests.py::TestShell::test_open - TypeError: argument of type 'int' is not iterable
FAILED tests/test_shell_tests.py::TestShell::test_shell_ping - RuntimeError: 1 PING www.google.com (142.251.214.132) 56(84) bytes of data.

--- www.google.com ping statistics ---
1 packets transmitted, 0 received, 100% packet loss, time 0ms
FAILED tests/ssh/test_ssh.py::TestSsh::test_ssh_processors - assert False
======= 7 failed, 82 passed, 1 skipped, 10 warnings in 109.67s (0:01:49) =======
Error: Process completed with exit code 1.

--- a/.github/workflows/python-package-conda.yml
+++ b/.github/workflows/python-package-conda.yml
@@ -85,4 +85,4 @@ jobs:
     - name: Test with pytest
       run: |
         conda install pytest
-        source activate base & pytest tests
+        source activate base & pytest -s tests

--- a/cloudmesh/common/Host.py
+++ b/cloudmesh/common/Host.py
@@ -328,7 +328,8 @@ class Host(object):
         count_flag = '-n' if platform == 'windows' else '-c'
         command = ['ping', count_flag, count, ip]
         result = subprocess.run(command, capture_output=True)
-
+        print(result)
+        print('THAT WAS RESULT!!!')
         try:
             timers = result.stdout \
                 .decode("utf-8", "ignore") \

--- a/cloudmesh/common/Host.py
+++ b/cloudmesh/common/Host.py
@@ -11,6 +11,7 @@ from cloudmesh.common.DateTime import DateTime
 from cloudmesh.common.parameter import Parameter
 from cloudmesh.common.util import path_expand
 from cloudmesh.common.util import readfile
+from cloudmesh.common.systeminfo import os_is_windows
 from pprint import pprint
 from cloudmesh.common.Printer import Printer
 
@@ -325,8 +326,15 @@ class Host(object):
             """
         ip = args['ip']
         count = str(args['count'])
-        count_flag = '-n' if platform == 'windows' else '-c'
-        command = ['ping', count_flag, count, ip]
+        
+        count_flag = '-n' if os_is_windows() else '-c'
+        if os_is_windows():
+            # adding ipv4 enforce for windows
+            # for some reason -4 is required or hosts
+            # fail. we need ipv4
+            command = ['ping', '-4', ip, count_flag, count]
+        else:
+            command = ['ping', count_flag, count, ip]
         result = subprocess.run(command, capture_output=True)
         print(result)
         print('THAT WAS RESULT!!!')

